charts/aws-vpc-cni/values.yaml

```diff
@@ -57,6 +57,7 @@ env:
   DISABLE_NETWORK_RESOURCE_PROVISIONING: "false"
   ENABLE_IPv4: "true"
   ENABLE_IPv6: "false"
+  IP_COOLDOWN_PERIOD: "30"
```

Comment:
No entry is needed here, as this is not a mandatory environment variable. Let's revert all changes in this file